frontend/src/api/WorkWithMydata.test.ts

```diff
@@ -0,0 +1,17 @@
+import {HttpClient} from './HttpClient'
+import {WorkWithMyData} from './WorkWithMyData'
+
+jest.mock('./HttpClient')
+const httpClient = new HttpClient()
+
+describe('use of WorkWithMyData client', () => {
+  beforeEach(() => {
+    HttpClient.mockClear()
+  })
+
+  it('WorkWithMyData should be able to call new() on HttpClient', () => {
```

Comment:
This test does not actually testing what is described.
You need to test that getData and saveData are make proper calls to HttpClient